### DISCUSSION TITLE
Dropdown for select field in index view

### DISCUSSION THREAD

**Discussion by gervic08** [avo-hq/avo#1672] [Help] [answered]
Hello, I need to know if there is an approach to make available a select field in the index view in order to ease the multiple assignation of associated attriubutes without having to enter one by one to their show view.

![dropdowns](https://user-images.githubusercontent.com/92609839/230801103-7e3f3fad-b636-423a-84db-e1a1f17db5cf.png)

**Comment by FLX-0x00** [comment, marked as answer]
This may not be the right position. The index view is a view by design, not a form. I think the correct approach is a custom action. That way you can use the index view to select multiple items and assign options in a drop down contained in the custom action.

**Reply by adrianthedev** [reply]
Yeah, those were our thoughts too when we designed the index view.

**Reply by gervic08** [reply]
Thank you guys for the response! It helps a lot to me to clarify the way I have to approach that.

**Comment by adrianthedev** [comment]
I do see a plus in having that there sometimes. Avo 3 will have customizable row controls baked in by default.

Until then you can generate a [custom field](https://docs.avohq.io/2.0/custom-fields.html) which will give you access to the index view component where you can add your own view logic and you'll be able to add everything you need.

**Reply by FLX-0x00** [reply]
I'm already afraid to read the gigantic documentation for version 3 xD hopefully we can train GPT4 on it
